src/dfu.rs

```diff
@@ -718,7 +718,7 @@ impl<const MTU: usize> DfuTarget<MTU> {
                 obj.crc.add(data);
                 obj.offset += data.len() as u32;
 
-                let mut response = DfuResponse::new(request, DfuResult::Success);
+                let mut response = DfuResponse::new(DfuRequest::Crc, DfuResult::Success);
```

Comment:
I'm not sure I understand this change. The first parameter here is the originating request, the Crc is set in the body: https://github.com/lulf/nrf-dfu/blob/main/src/dfu.rs#L455-L463 Changing the request type to be Crc instead of Write doesn't make sense to me. Are you seeing a bug occuring without this change?

Reply:
Ok, thanks for confirming @plaes !